src/utils/connection.js

```diff
@@ -8,7 +8,7 @@ let collection = {};
 
 collection.connectDB = async () => {
 	try {
-		await mongoose.connect(url, {
+		await mongoose.connect(process.env.MONGOLAB_URI || url, {
```

Comment:
while declaring the **url** at **line 3** i have already declared the have to be used from env variable. so need to do it again here.

Note: Use git commit --amend after the change so as to not create another commit for the same.